Protocol.md

```diff
@@ -185,7 +191,8 @@ and closing transactions.
    * [`u64`:`change_serial_id`]
    * [`cet_adaptor_signatures`:`cet_adaptor_signatures`]
    * [`signature`:`refund_signature`]
-   * [`negotiation_fields`:`negotiation_fields`]
+   * [`negotiation_fields`:`negotiation_fields`] (Optional: 1)
+   * [`accept_tlvs`: `tlvs`]
```

Comment:
`bigsize` here too i think https://github.com/discreetlogcontracts/dlcspecs/pull/163/files#diff-e0f5b925f91a1c09c6daf26f9a7d28816cb9ff9f08863faca719b7ee0a1cc065R188

Reply:
Sorry not sure which field you're referring to in this comment?

Reply:
`num_funding_inputs`

Reply:
Thanks!